test/test.js

```diff
@@ -0,0 +1,18 @@
+import chai from 'chai'
+import app from '../src/main';
+let assert = chai.assert;
+
+describe('app to test as', () => {
+    let result = app();
+    it('app should return hello', () => {
+
+       assert.equal(result, 'hello')
+    });  // we put app() becos we checking for app()----goto terminal type npm run test
+
+   it('it should be a string', () => {
+        assert.typeOf(result, 'string')
+    });
+});
```

Comment:
You wrote tests but your tests are not running because you didn't update your package.json with the test script